contributor-guide/modules/ROOT/pages/version-control.adoc

```diff
@@ -1,4 +1,138 @@
+
+As with any Git-based project, you can clone the repositories, make changes, commit those changes to your local repository, and push your changes back to the server. If you want to contribute changes to the official Boost libraries, refer to xref:getting-involved.adoc#contribute[Contribute to an Existing Library].
+
+== The Super-project Repository
+
+The Boost https://github.com/boostorg/boost[*Super-project*] repository includes all the libraries as submodules.
+
+In addition to the libraries, you'll find many other repositories under `boostorg` that handle other tasks, for example:
+
+* https://github.com/boostorg/website2022[Website generation]
+* https://github.com/boostorg/cmake[CMake support]
+* https://github.com/boostorg/regression[Regression testing]
+* https://github.com/boostorg/release-tools[Release tools]
+
+CMake is one of the most popular of the https://github.com/boostorg/boost/tree/master/tools[boost/tools] directory. You will find other tools used for tasks such as building and testing Boost libraries. 
+
+The Super-project has both *master* and *develop* branches, which operate somewhat differently than for individual libraries. 
+
+New features are added and bugs are fixed in library *develop* branches. When the *develop* branch is complete, it is merged with the library *master* branch. 
+
+For the Super-project, the *develop* and *master* branches are independent. Both branches track the latest changes in the corresponding library branch, for all libraries. The Super-project branches are never merged, as this strategy prevents merging. In other words, the Super-project *master* branch is created from the library *master* branches, not from the Super-project *develop* branch (which is primarily used for testing).
+
+When there is a public release of Boost, it is built from the *master* branch after that branch has been closed to updates (refer to the xref:release-process.adoc[]).
+
+=== Working with the Super-project
+
+You might consider creating submodules for a project, when developing your library. This is not a required process, but it does fit with some of the Super-project tooling:
+
+* In Git, when you add a submodule to a project, you make it point to a particular commit. This means that, when developers update their sub-projects, the Super-project _doesn't_ get immediately updated. For this reason, there is a commit bot that runs nightly and updates the commit to which each submodule references in the Super-project. This happens nightly and for both the *develop* and *master* branches. 
+
+* The implications of merging something in your library or submodule to the Super-project *develop* branch, include that due to how Boost is tested, you may affect other developers. As part of Boost testing, Continuous Integration (CI) usually clones the Super-project and any required submodules. 
```

Comment:
You don't merge to the super-project develop branch. You merge to your project's develop branch, and the commit bot will update the super-project.

Reply:
OK, got it